pandas/tests/test_resample.py

```diff
@@ -3274,3 +3274,17 @@ def test_aggregate_with_nat(self):
 
             # if NaT is included, 'var', 'std', 'mean', 'first','last'
             # and 'nth' doesn't work yet
+
+    def test_scalar_call_versus_list_call(self):
+        data_frame = pd.DataFrame({
```

Comment:
this will not lint properly

add a comment with the issue number.

this goes in the groupby tests

Reply:
actually ok with the file location of the tests, but there is a section in test_resample.py that does a groupby & a resample, pls put after that

Reply:
which test? I think put it in TestTimeGrouper is ok?